apps/src/code-studio/initApp/loadApp.js

```diff
@@ -479,10 +479,11 @@ const sourceHandler = {
       let source;
       let appOptions = getAppOptions();
       if (window.Blockly && Blockly.mainBlockSpace) {
+        const getSourceAsJson = true;
         // If we're readOnly, source hasn't changed at all
         source = Blockly.cdoUtils.isWorkspaceReadOnly(Blockly.mainBlockSpace)
           ? currentLevelSource
-          : Blockly.cdoUtils.getCode(Blockly.mainBlockSpace);
+          : Blockly.cdoUtils.getCode(Blockly.mainBlockSpace, getSourceAsJson);
```

Comment:
If any problems are found after merging, we can fix forward, (ie. stop saving as JSON anywhere) by removing `getSourceAsJson` here.

Reply:
I know we're feeling confident here, but feels to me like a big enough change to warrant a DCDO flag controlling `getSourceAsJson`?

Reply:
Thanks for the suggestion! Just added a DCDO flag.